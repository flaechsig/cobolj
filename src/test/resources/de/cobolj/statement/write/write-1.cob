IDENTIFICATION DIVISION.
PROGRAM-ID. HELLO.

ENVIRONMENT DIVISION.
   INPUT-OUTPUT SECTION.
   FILE-CONTROL.
   SELECT STUDENT ASSIGN TO 'test-output/write/write-1.txt'.

DATA DIVISION.
   FILE SECTION.
   FD STUDENT.
   01 STUDENT-FILE.
      05 STUDENT-ID PIC 9(5).
      05 NAME PIC A(25).
      05 CLASSNAME PIC X(3).

PROCEDURE DIVISION.
   OPEN OUTPUT STUDENT.
   MOVE 1000 TO STUDENT-ID.
   MOVE 'Tim' TO NAME.
   MOVE '10' TO CLASSNAME.
   WRITE STUDENT-FILE.	
   CLOSE STUDENT.
   
   OPEN INPUT STUDENT.
     READ STUDENT
         AT END DISPLAY 'End of Records'
         NOT AT END DISPLAY STUDENT-FILE
     END-READ
   CLOSE STUDENT.
STOP RUN.