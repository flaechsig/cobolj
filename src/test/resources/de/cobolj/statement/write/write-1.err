Fehler beim Schreiben einer Datei