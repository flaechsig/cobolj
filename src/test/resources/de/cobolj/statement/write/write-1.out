01000Tim                      10 
End of Records
